test/test_tools_add_pre_post_processing_to_model.py

```diff
@@ -320,6 +320,28 @@ def test_debug_step(self):
 
         self.assertEqual(len(new_model.graph.output), len(input_model.graph.output) + len(post_processing))
 
+    def test_make_border_and_draw_box(self):
```

Comment:
This a great start but I think there's a lot more test coverage needed.

Some test cases that comes to mind
- boxes that share a border with the image (need to test top, bottom, left and right
- box that goes off image (e.g. user did something wrong but we should still draw the part of the box that is valid for the image)
- colour by classes with more classes than colours
- colour by score with more results than colours
- overlapping boxes
  - how do the colours look? should we for example figure start drawing the lowest scoring results once we determine the full set of boxes to draw so that the lines for the best results are done last and are on top?

Reply:
> overlapping boxes

Yeah, The higher-score box will be drawn latter and has better visibility.